.rubocop.yml

```
@@ -5,6 +5,7 @@ inherit_gem:
 
 AllCops:
   TargetRubyVersion: 3.0
```

This should be set to the lowest version of Ruby we support with this gem, i.e. 2.7
(See https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version)
```suggestion
  TargetRubyVersion: 2.7
```

---

> Looks good. Would it be worth adding in the suggested Rubocop extensions in this PR too (rubocop-rake and rubocop-rails), or would that be better in a separate one?

Probably best to split out. not sure they add much value to this if im honest